src/build/build-bach.ss

```diff
@@ -125,13 +125,15 @@
           ["-e" include-gambit-sharp]))
        (output-bin
         (path-expand "gerbil" gerbil-bindir))
-       (gerbil-shared?
-        (member "--enable-shared" (string-split (configure-command-string) #\')))
+       (configure-flags (string-split (configure-command-string) #\'))
+       (gerbil-shared? (member "--enable-shared" configure-flags))
+       (enable-openssl? (member "--enable-openssl" configure-flags))
```

Comment:
Please dont!

We do not enable-openssl for gambit at all and should not use this; furthermore, openssl is a hard requirement for gerbil amd not something to allow turning off.


In fact we might want to filter this optiin out, but it might be useful for people who want to use ssl with gambit ports